ospackage/software_recipes/file_handler.go

```diff
@@ -0,0 +1,86 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package software_recipes
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	"./oswrap"
```

Comment:
This should be the fill import path, but since you only use a few functions I would just copy them into this package, we can decide later if we need a shared package.